src/common/format/common.fbs

```diff
@@ -49,4 +49,22 @@ table TaskInfo {
   required_resources: [double];
 }
 
+// Object information data structure.
+table ObjectInfo {
+  // Object ID of this object.
+  object_id: string;
+  // Number of bytes the content of this object occupies in memory.
+  data_size: long;
+  // Number of bytes the metadata of this object occupies in memory.
+  metadata_size: long;
+  // Unix epoch of when this object was created.
+  create_time: long;
+  // How long creation of this object took.
+  construct_duration: long;
+  // Hash of the object content.
+  digest: string;
+  // Specifies if this object was deleted or added.
+  is_deletion: bool;
```

Comment:
Yeah, that's right. We had it as part of the original object_info and it is easier to not have another message that's why I kept it.